external/loaders/loaders/mappers/_fine_res.py

```diff
@@ -41,24 +41,31 @@ def standardize_coords(
     return gfdl_to_standard(ds_shifted).drop("tile")
 
 
-def _open_merged_dataset(
-    fine_url: str, additional_dataset_urls: Optional[Sequence[str]]
-) -> FineResBudget:
-
-    fine = open_zarr(fine_url)
-    fine_shifted = standardize_coords(fine)
+def _merge(fine, additional_dataset_urls):
 
     if additional_dataset_urls is not None:
         additional_datasets = []
         for url in additional_dataset_urls:
             additional_datasets.append(open_zarr(url))
-        merged = xr.merge([fine_shifted, *additional_datasets], join="inner")
+        merged = xr.merge([fine, *additional_datasets], join="inner")
         if "latitude" in merged:
             merged["latitude"] = merged.latitude.isel(time=0)
         if "longitude" in merged:
             merged["longitude"] = merged.longitude.isel(time=0)
     else:
-        merged = fine_shifted
+        merged = fine
+
+    return merged
+
+
+def _open_merged_dataset(
+    fine_url: str, additional_dataset_urls: Optional[Sequence[str]]
+) -> FineResBudget:
+
+    fine = open_zarr(fine_url)
+    fine_shifted = standardize_coords(fine)
+
+    merged = _merge(fine_shifted, additional_dataset_urls)
 
     # enforce that these ML inputs come from fine dataset
     merged["air_temperature"] = fine_shifted.T
```

Comment:
should this not be part of the `_merge` func so it happens for both cases?

Reply:
Was actually trying to exclude this since the offline datasets I've made don't tend to have T and sphum (just Q1 and Q2).

Reply:
Okay, maybe add some if statements to check for existence of `T` and `sphum`? I think code will be easier to follow if both mappers use the same function.